### PR TITLE
Fixed iOS not handling cookies properly

### PR DESCRIPTION
**Pull request by illegalDisease**
Added a handler event to hold cookies after navigation on iOS and a function to parse cookies.

### CODE REVIEW

**Comment by ozcanovunc** [comment]
Dogan developed a solution for maintaining the cookie across web pages. And claims that, existing [request](https://github.com/smartface/sf-extension-utils/pull/17/files#diff-999dfa57ce515aa34a3f785a0e4f9917L423) must be performed along with onStartDocumentHandler method. It gets triggered when document is about to load so when a web page is being navigated to, document.cookie is set with that callback.

File: lib/webviewbridge.js [hunk 1]
```diff
@@ -405,6 +405,9 @@ const WebViewBridge = new extend(EventEmitter)(function(_super, options) {
 
 });
 
+function parseCookie(cookie) {
+    return cookie.split(";").map(comp => "document.cookie='" + comp + "'").join(";") + ";";
```
[the remark this answers is not in the document]
Comment (illegalDisease) [reply lib/webviewbridge.js]: Yes, whenever we request the cookie of a token, the service responds with a different cookie. You might say that it is tested with plenty of cookies.

[the remark this answers is not in the document]
Comment (illegalDisease) [reply lib/webviewbridge.js]: Done. To double check, matched both snippets with two different cookies, and results were the same. === operator also returned true in both cases.

File: lib/webviewbridge.js [hunk 2]
```diff
@@ -414,6 +417,11 @@ function customNavigate(options) {
 
 
     if (System.OS === "iOS") {
+        if (cookie) {
```
[the remark this answers is not in the document]
Comment (illegalDisease) [reply lib/webviewbridge.js]: Done.

**Comment by alperozisik** [comment]
@ozcanovunc @illegalDisease sf-extension-utils@4.4.1 has been released

**Pull request merged**